packages/config/config/devices/0x0129/templates/yale_template.json

```diff
@@ -331,5 +331,136 @@
 		"readOnly": false,
 		"writeOnly": false,
 		"allowManualEntry": true
+	},
+	"alarm_map_keypad_limit": {
+		"from": {
+			"alarmType": 161,
+			"alarmLevel": 1
+		},
+		"to": {
+			"notificationType": 6,
+			"notificationEvent": 16
+		}
+	},
+	"alarm_map_manual_unlock": {
+		"from": {
+			"alarmType": 22,
+			"alarmLevel": 1
+		},
+		"to": {
+			"notificationType": 6,
+			"notificationEvent": 2
+		}
+	},
+	"alarm_map_rf_unlock": {
+		"from": {
+			"alarmType": 25,
+			"alarmLevel": 1
+		},
+		"to": {
+			"notificationType": 6,
+			"notificationEvent": 4
+		}
+	},
+	"alarm_map_keypad_unlock": {
+		"from": {
+			"alarmType": 19
+		},
+		"to": {
+			"notificationType": 6,
+			"notificationEvent": 6
+		}
+	},
+	"alarm_map_manual_lock": {
+		"from": {
+			"alarmType": 21
+		},
+		"to": {
+			"notificationType": 6,
+			"notificationEvent": 1
+		}
+	},
+	"alarm_map_rf_lock": {
+		"from": {
+			"alarmType": 24,
+			"alarmLevel": 1
+		},
+		"to": {
+			"notificationType": 6,
+			"notificationEvent": 3
+		}
+	},
+	"alarm_map_keypad_lock": {
+		"from": {
+			"alarmType": 18
+		},
+		"to": {
+			"notificationType": 6,
+			"notificationEvent": 5
+		}
+	},
+	"alarm_map_deadbolt_jammed": {
+		"from": {
+			"alarmType": 9
+		},
+		"to": {
+			"notificationType": 6,
+			"notificationEvent": 11
+		}
+	},
+	"alarm_map_non_access": {
+		"from": {
+			"alarmType": 38
+		},
+		"to": {
+			"notificationType": 6,
+			"notificationEvent": 254
+		}
+	},
+	"alarm_map_low_battery": {
+		"from": {
+			"alarmType": 167
+		},
+		"to": {
+			"notificationType": 6,
+			"notificationEvent": 10,
+			"eventParameters": {
+				"userId": "alarmLevel"
+			}
+		}
+	},
```

Comment:
That looks incorrect. Why would low_battery be mapped to "Auto lock not fully locked operation" with a user ID?

Reply:
![Screen Shot 2021-03-13 at 12 01 11 AM](https://user-images.githubusercontent.com/33612110/111019766-479d8680-838f-11eb-8ea4-180af4860f24.png)

Maybe I'm confused on the userId part. It seemed off to me. In this case the alarmLevel is the battery %. How do I map that in?